src/views/Main/settings_mock.json

```diff
@@ -0,0 +1,8 @@
+{
+  "favicon": "<favicon>",
+  "title": "title",
+  "theme": { 
+    "logo": "<URL>",
+    "style": ["<URL>"]
+  }
+}
```

Comment:
Missing newline

Reply:
up

Reply:
Is settings_mock.json still useful?

Reply:
yes for deployment as long as the settings backend module is not activated.